spot-client/src/common/remote-control/remoteControlClient.js

```diff
@@ -487,6 +513,28 @@ export class RemoteControlClient extends BaseRemoteControlService {
             return;
         }
 
+        this._onSpotTvStatusReceived(from, state);
+    }
+
+    /**
+     * Spot TV status processing.
+     *
+     * @param {string} from - Spot TV address from which the status has update has been received.
+     * @param {Object} state - JSON object with TV's state.
+     * @private
+     * @returns {void}
+     */
+    _onSpotTvStatusReceived(from, state) {
+        // NOTE There will be no timestamp in the initial Spot TV presence
+        const { timestamp } = state;
+
+        const currentTimestamp = this._lastSpotState && this._lastSpotState.timestamp;
+
+        if (currentTimestamp && timestamp && currentTimestamp >= timestamp) {
```

Comment:
Something converts the timestamp to a number, correct? I'm thinking about the case where presence is in xml, not json, and the timestamp is somehow obtained from it.

Reply:
JSON.parse coverts it on the presence to XML conversion. I don't understand the XML case, when could it happen?